File: platform/sdk/python/examples/graph/graph-027_ac15_f8_cb_plus_audit.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.ac15F8CbPlusAudit()
if not result["passed"] or not result["callbackInvoked"]:
    sys.exit(1)
print("XJ-OK: graph-027_ac15_f8_cb_plus_audit")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-027_ac15_f8_cb_plus_audit.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.ac15F8CbPlusAudit();
if (!result.passed || !result.callbackInvoked) process.exit(1);
console.log("XJ-OK: graph-027_ac15_f8_cb_plus_audit");
process.exit(0);
